server/index.js

```diff
@@ -8,6 +9,10 @@ const app = express();
 app.use(express.static('./public'));
 app.use(cors());
 
+function getRandomInt(max) {
```

Comment:
This function should live in its own file referenced by an `artillery.yml` file, not one run by your server process.